doc/level1.rst

```diff
@@ -682,12 +685,15 @@ this case contains one Autocrypt-Gossip header for each recipient,
 each of which:
 
 - MUST include an ``addr`` attribute that matches one of the
-  recipients in the ``To`` or ``Cc`` headers.
+  recipients in the ``To``, ``Cc``, or ``Reply-To`` headers.
 
-- MUST include the ``keydata`` attribute which MUST contain the
+- MUST include the ``keydata`` attribute.
+  For ``To`` and ``Cc`` headers it MUST contain the
   same public key which is used to encrypt the mail to the recipient
   referenced by ``addr``. See also :ref:`preliminary recommendation`
   for how this key is selected.
+  For ``Reply-To`` headers it MUST contain the public key
+  which should be used to encrypt a reply.
```

Comment:
what if a sender has the keys for To/CC recipients (which it must in order to encrypt) but not for Reply-To?

Reply:
this is per individual header, so in that case there simply wouldn't be a header with an `addr` to match the `Reply-To`

Reply:
I'm not convinced that MUST is particularly meaningful in this new sentence -- how could it be contradicted?  I'm also pedantically concerned that it seems to imply that any reply (even those that aren't addressed to the `Reply-To` address) should be encrypted to that key.